LoopbackGuideApplication/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.10.0'
+        classpath 'com.android.tools.build:gradle:1.5.+'
```

Comment:
Gradle is updated to 2.10
That's the Android plugin for Gradle. The most recent version is 1.5
I got confused with that before till I got an error in the build.